python/rascaline-torch/rascaline/torch/calculator_base.py

```diff
@@ -97,20 +103,32 @@ def compute(
             ``None`` or an empty list ``[]``, no gradients are kept in the output. Some
             gradients might still be computed at runtime to allow for backward
             propagation.
+
+        :param use_native_system: If :py:obj:`True` (this is the default), copy data
+            from the ``systems`` into Rust ``SimpleSystem``. This can be a lot
+            faster than having to cross the FFI boundary often when accessing
+            the neighbor list. :py:obj:`False` is not supported.
```

Comment:
```suggestion
        :param use_native_system: This can only be :py:obj:`True`, and is here for compatibility 
            with the same parameter on :py:meth:`rascaline.calculators.CalculatorBase.compute`.
```